components/navigation/helpers.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+import { StyledAnchor } from '@components/top-bar/styled'
+import { NavLink } from '@common/constants'
+
+export const ErrorPageLink: React.FC<any> = ({ isErrorPage, children, href, ...props }) => (
```

Comment:
Nit: Can we make an explicit interface for these props? It would be great to not have explicity `any`s as we migrate to TS.